sql/4_2_2-to-5_0_0_upgrade.sql

```diff
@@ -713,6 +713,10 @@ INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `notes`, `ac
 #IfMissingColumn procedure_order history_order
 ALTER TABLE procedure_order ADD COLUMN history_order enum('0','1') DEFAULT '0';
 #EndIf
+ 
+#IfMissingColumn form_encounter pos_code
+       ALTER TABLE `form_encounter` ADD `pos_code` INT(2) NOT NULL ;
```

Comment:
In facility table, the following was used(does it make more sense to use the same?):
``````
`pos_code` tinyint(4) default NULL